tox.ini

```diff
@@ -1,14 +1,21 @@
 [tox]
 envlist =
-    py26,py27,py33,py34
+    py27,py34,py35,py36,py37,coverage
 
 [testenv]
+usedevelop = true
```

Comment:
Why? Do tests fail if you don't do this?

Reply:
I think it renders the path to the `src` directory if a test fails instead of the path to the `site-packages` inside the `.tox` directory but I might be wrong.

Reply:
I tried to omit `usedevelop = true` this makes coverage fail. I tells that no code is covered.
But after changing `.coveragerc` everything went fine again.
Only the paths in the coverage report look weird because they point into the `.tox` directory.

Reply:
I understand the advantages of `usedevelop` in `[testenv:coverage]`, but I was asking about the generic `[testenv]`.

(Yeah, the .tox/.../site-packages/... paths in tracebacks are annoying.  I've kept making fixes to the wrong files that got overwritten by the next tox run many times.)